### ISSUE TITLE
請問正確的路徑配置?

### ISSUE THREAD

**Issue by BenoitKAO** [JaminFong/DenseNAS#22]
感謝您們釋出這麼棒的研究工作!

在使用你們模組時遇到困難,
想請問,
Search區塊裡:

1.Prepare the image set for search which contains 100 classes of the original ImageNet dataset. And 20% images are used as the validation set and 80% are used as the training set.

1). Generate the split list of the image data.
python dataset/mk_split_img_list.py --image_path 'the path of your ImageNet data' --output_path 'the path to output the list file'

2). Use the image list obtained above to make the lmdb file.
python dataset/img2lmdb.py --image_path 'the path of your ImageNet data' --list_path 'the path of your image list generated above' --output_path 'the path to output the lmdb file' --split 'split folder (train/val)''
...

詳盡的路徑配置和各個floder裡面包含的架構與內容?

從ImageNet下載datasets,
經過各種設置的嘗試,
都沒有成功.

謝謝.

**Comment by BenoitKAO** [comment]
img2lmdb.py裡,

發現
這一行
image_folder = osp.join(args.image_path, args.split)
應該要更改成
image_folder = osp.join(args.image_path, args.image_path)
